src/math/calculation.js

```diff
@@ -28,6 +28,8 @@ var p5 = require('../core/core');
  * </code></div>
  */
 p5.prototype.abs = Math.abs;
+// Avoid `.bind`ing this method in global mode
+p5.prototype.abs._context = false;
```

Comment:
Before we commit to this approach I would like several people to thoroughly test whether `abs` in this branch is faster than `abs` in master (both in global mode), as well as the same test for a custom-implementation function like `random`.